packages/tocco-ui/src/Typography/StyledList.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components'
+import {theme} from 'styled-system'
+
+import {
+  declareFont,
+  declareNoneWrappingText,
+  declareWrappingText
+} from '../utilStyles'
+
+const StyledDd = styled.dd`
+  && {
+    ${props => declareFont(props)}
+    ${props => props.breakWords ? declareWrappingText() : declareNoneWrappingText()}
+    margin: 0;
+  }
+`
+
+const StyledDl = styled.dl`
+  && {
+    margin: 0 0 ${props => theme('space.5')};
+  }
+`
+
+const StyledDt = styled.dt`
+  && {
+    ${props => declareFont(props, {fontWeight: 700})}
+    ${props => props.breakWords ? declareWrappingText() : declareNoneWrappingText()}
+    margin: 0;
+  }
+`
+
+const StyledLi = styled.li`
+  && {
+    ${props => declareFont(props)}
+    ${declareWrappingText()}
+  }
+`
+
+const StyledList = styled.ol`
+  && {
+    display: block;
+    list-style-position: outside;
+    margin: 0 0 ${props => theme('space.5')} 1.6rem;
+    padding: 0;
+
+    & & {
```

Comment:
same as && ?

Reply:
'&&' is not equal '& &'
* '&&' => control specificity (https://www.styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity)
* '& &' => control nesting  (not exactly https://www.styled-components.com/docs/advanced#referring-to-other-components but has the same goal)